test/unit/__snapshots__/Incentives.spec.ts.snap

```diff
@@ -1,5 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`unit/Incentives #createIncentive works and has gas cost 1`] = `59981`;
+exports[`unit/Incentives #createIncentive works and has gas cost 1`] = `57510`;
```

Comment:
due to not sloading the second slot of an incentive that is known to be zero before an incentive is created